src/components/ComboChartCard/ComboChartCard.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ComboChart from '@carbon/charts-react/combo-chart';
+import classnames from 'classnames';
+import isEmpty from 'lodash/isEmpty';
+
+import { CARD_SIZES } from '../../constants/LayoutConstants';
+import Card from '../Card/Card';
+import { CardPropTypes } from '../../constants/CardPropTypes';
+import { settings } from '../../constants/Settings';
+
+const { iotPrefix } = settings;
+
+const propTypes = {
+  id: PropTypes.string.isRequired,
+  i18n: PropTypes.shape({
```

Comment:
We'll need the same i18n as TimeSeriesCard and BarChartCard I think as we translate as many things as possible